codegenerator/model/api.go

```diff
@@ -74,20 +75,12 @@ import org.mozilla.taskcluster.client.TaskClusterRequestHandler;
 
     protected static final String defaultBaseURL = "` + api.BaseURL + `";
 
-    public ` + className + `(String clientId, String accessToken) {
-        super(clientId, accessToken, defaultBaseURL);
```

Comment:
Well, *we* are not using it -- we don't actually know if anyone else is using it.  I'll admit, it's unlikely, but it seems like this is an easy place to keep compatibility and avoid making some lonely Java programmer sadder than s/he already is..